### PR TITLE
BAYCKRC 900/2400 Dual Band 1W Nano Gemini TX

### PR DESCRIPTION
**Pull request by JyeSmith**

### CODE REVIEW

**Review by pkendall64 — APPROVED** [review]

**Review by SunjunKim — APPROVED** [review]
Production sample are arrived, and the target configurations are confirmed.

**Review by SunjunKim — APPROVED** [review]

**Review by schugabe — APPROVED** [review]

File: targets.json [hunk 1]
```diff
@@ -113,6 +113,32 @@
...
+                "lua_name": "B 9002G4 NanoGTX",
+                "layout_file": "BAYCKRC Dual Band.json",
+                "overlay": {
+                    "radio_busy_2": 39,
+                    "radio_dio1_2": 34,
+                    "radio_nss_2": 15,
+                    "radio_rst_2": 21
+                },
+                "upload_methods": ["uart", "wifi"],
+                "min_version": "3.4.0",
```
Comment (mha1) [inline targets.json:137]: Shouldn't that also be 3.5.2?

Comment (JyeSmith) [reply targets.json:137]: This is fine as GemX functionality was added earlier

**Review by mha1 — APPROVED** [review]

**Review by deadbytefpv — APPROVED** [review]
I have the test samples. Quick connection checks are okay.

**Pull request merged**